### DISCUSSION TITLE
How can we know the outcome of our submission?

### DISCUSSION THREAD

**Discussion by hamzamogni** [geeksblabla/blanat#194] [Q&A]
I am assuming that when a submission is merged but not added to the leaderboard it means the run failed. But is there a way we can know why it failed? (wrong answer, timeout...etc) ?

**Comment by aboullaite** [comment]
We cant provide details on every submission sorry. If it doesn't show up, then the result was wrong or the program took longer than 15min

**Reply by hamzamogni** [reply]
Thanks for your answer :pray:  It is weird since when I test my code on a GCP VM with same specs (8 Vcores and 32G RAM) it takes ~3min to run on a 1 billion lines csv. Running the code on the example now yields the exact same output file.

Thank you again
